composer/core/time.py

```diff
@@ -647,6 +664,8 @@ def get(self, unit: Union[str, TimeUnit]) -> Time[int]:
             return self.sample
         if unit == TimeUnit.TOKEN:
             return self.token
+        if unit == TimeUnit.SECOND or unit == TimeUnit.MINUTE or unit == TimeUnit.HOUR:
+            return Time(int(self._total_wct.total_seconds()) if self._total_wct else 0, TimeUnit.SECOND)
```

Comment:
im confused, you always return seconds even if they ask for minutes or hours...?

Reply:
Same as above, though definitely can ask for a wider opinion

Reply:
my other comments makes this a bit complicated...would we start returning a timedelta here? or always convert to seconds? not super sure...